### ISSUE TITLE
ValueError: mutable default <class 'hydra.conf.JobConf.JobConfig.OverrideDirname'> for field override_dirname is not allowed: use default_factory

### ISSUE THREAD

**Issue by shrxht** [facebookresearch/co-tracker#85]
when I want to evaluate on TAP-Vid DAVIS, ```import hydra``` will raise this issue, how can I solve it

**Comment by nikitakaraevv** [comment]
Hi @shrxht, did you follow the installation instructions?
https://github.com/facebookresearch/co-tracker?tab=readme-ov-file#evaluation

That's how you should install hydra for the current version of code:`pip install hydra-core==1.1.0 mediapy`

**Comment by YannickGibson** [comment]
Solved the issue by downgrading from `Python/3.11.3-GCCcore-12.3.0` to `Python/3.10.4-GCCcore-11.3.0`.

Complete configuration:
```sh
module load GCCcore/11.3.0
module load CUDA/11.3.0
module load Python/3.10.4-GCCcore-11.3.0
module load OpenSSL/1.1
module load git/2.36.0-GCCcore-11.3.0-nodocs

source my_virtualenv/bin/activate
```

**Comment by YannickGibson** [comment]
Just to add, all tested python versions for the `hydra` package are `["3.6", "3.7", "3.8", "3.9"]` as denoted in: https://github.com/facebookresearch/hydra/blob/2808e71248ac5a04a1b1a770d3a60f8ec9a38569/noxfile.py#L15
